tardis/montecarlo/tests/test_cmontecarlo.py

```diff
@@ -426,7 +429,23 @@ def test_montecarlo_line_scatter(packet_params, expected_params, packet, model,
     assert_almost_equal(packet.tau_event, expected_params['tau_event'])
     assert_almost_equal(packet.next_line_id, expected_params['next_line_id'])
 
+"""
+Simple Tests:
+----------------
+These test check very simple pices of code still work.
+"""
+
+@pytest.mark.parametrize(
+    ['packet_params', 'line_idx', 'expected'],
+    [({'energy':0.0}, 0, 0),
+     ({'energy':1.0}, 1, 1),
+     ({'energy':0.5}, 2, 1.5)]
+)
```

Comment:
I know the parametrization in the code is currently done mostly this way but I think we should do new tests differently.
Have a look at https://github.com/yeganer/tardis/blob/4511369c245b6a6fb9a33a6891089e20efac56a3/tardis/montecarlo/tests/test_cmontecarlo.py#L282 to see how I wrote the tests.